src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/InputIllegalTypeSameFileNameFalsePositive.java

```diff
@@ -19,13 +19,11 @@
 import java.util.ArrayList;
 import java.util.Date;
 
-import com.puppycrawl.tools.checkstyle.checks.coding.illegaltype.InputIllegalTypeGregCal.SubCal;
-
```

Comment:
This is very special class thwr keep in our inputs
https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/InputIllegalTypeGregCal.java
To allow to catch specific use case.

Please add this Input file and one Input below to permanent suppression, as I suggested in some other PR.

Reply:
done